platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -155,7 +156,13 @@ public T getClientForSubject(Subject subject) throws SecurityServiceException {
      * @see #getClientForSubject(Subject subject)
      */
     public WebClient getWebClientForSubject(Subject subject) throws SecurityServiceException {
-        return WebClient.fromClientObject(getClientForSubject(subject));
```

Comment:
`return WebClient.fromClient(WebClient.client(getClientForSubject(subject)), true);` should work here since getClientForSubject will always return a new client.